test/sanitizer_suppressions/ubsan

```diff
@@ -66,6 +66,7 @@ implicit-integer-sign-change:verify_flags
 implicit-integer-sign-change:EvalScript
 implicit-integer-sign-change:serialize.h
 implicit-signed-integer-truncation:crypto/
+implicit-signed-integer-truncation,implicit-integer-sign-change:secp256k1_modinv64_posdivsteps_62_var
```

Comment:
Shouldn't this be covered by https://github.com/hebasto/bitcoin/blob/e39bae122c79b8dfaa5f7e02ff199dc8c2051d8a/test/sanitizer_suppressions/ubsan#L22-L28 already?

Reply:
It expects exactly "implicit-signed-integer-truncation,implicit-integer-sign-change".

Reply:
I see. I think it still makes sense to move the line up so that it appears below `# Suppressions in dependencies that are developed outside this repository.`

Reply:
Sure. Done.